src/client/components/SearchAPI/Fetcher.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
```

Comment:
SearchAPI is not a component. But Fetcher might be a component so `components/Fetcher/Fetcher`